zetasql-toolkit-bigquery/src/main/java/com/google/zetasql/toolkit/options/BigQueryLanguageOptions.java

```diff
@@ -25,150 +25,197 @@
+                        LanguageFeature.FEATURE_V_1_1_HAVING_IN_AGGREGATE,
+                        LanguageFeature.FEATURE_V_1_1_LIMIT_IN_AGGREGATE,
+                        LanguageFeature.FEATURE_V_1_1_NULL_HANDLING_MODIFIER_IN_AGGREGATE,
+                        LanguageFeature.FEATURE_V_1_1_NULL_HANDLING_MODIFIER_IN_ANALYTIC,
+                        LanguageFeature.FEATURE_V_1_1_ORDER_BY_IN_AGGREGATE,
+                        LanguageFeature.FEATURE_V_1_1_SELECT_STAR_EXCEPT_REPLACE,
+                        LanguageFeature.FEATURE_V_1_1_WITH_ON_SUBQUERY,
+                        LanguageFeature.FEATURE_V_1_2_CIVIL_TIME,
+                        LanguageFeature.FEATURE_V_1_2_SAFE_FUNCTION_CALL,
+                        LanguageFeature.FEATURE_V_1_2_WEEK_WITH_WEEKDAY,
+                        LanguageFeature.FEATURE_V_1_3_ADDITIONAL_STRING_FUNCTIONS,
+                        LanguageFeature.FEATURE_V_1_3_ALLOW_DASHES_IN_TABLE_NAME,
+                        LanguageFeature.FEATURE_V_1_3_ALLOW_REGEXP_EXTRACT_OPTIONALS,
+                        LanguageFeature.FEATURE_V_1_3_ANNOTATION_FRAMEWORK,
+                        LanguageFeature.FEATURE_V_1_3_CASE_STMT,
+                        LanguageFeature.FEATURE_V_1_3_COLLATION_SUPPORT,
+                        LanguageFeature.FEATURE_V_1_3_COLUMN_DEFAULT_VALUE,
+                        LanguageFeature.FEATURE_V_1_3_CONCAT_MIXED_TYPES,
+                        LanguageFeature.FEATURE_V_1_3_DATE_ARITHMETICS,
+                        LanguageFeature.FEATURE_V_1_3_DATE_TIME_CONSTRUCTORS,
+                        LanguageFeature.FEATURE_V_1_3_DECIMAL_ALIAS,
+                        LanguageFeature.FEATURE_V_1_3_EXTENDED_DATE_TIME_SIGNATURES,
+                        LanguageFeature.FEATURE_V_1_3_EXTENDED_GEOGRAPHY_PARSERS,
+                        LanguageFeature.FEATURE_V_1_3_FORMAT_IN_CAST,
+                        LanguageFeature.FEATURE_V_1_3_FOR_IN,
+                        LanguageFeature.FEATURE_V_1_3_IS_DISTINCT,
+                        LanguageFeature.FEATURE_V_1_3_LIKE_ANY_SOME_ALL,
+                        LanguageFeature.FEATURE_V_1_3_NULLS_FIRST_LAST_IN_ORDER_BY,
+                        LanguageFeature.FEATURE_V_1_3_OMIT_INSERT_COLUMN_LIST,
+                        LanguageFeature.FEATURE_V_1_3_PIVOT,
+                        LanguageFeature.FEATURE_V_1_3_QUALIFY,
+                        LanguageFeature.FEATURE_V_1_3_REMOTE_FUNCTION,
+                        LanguageFeature.FEATURE_V_1_3_REPEAT,
+                        LanguageFeature.FEATURE_V_1_3_SCRIPT_LABEL,
+                        LanguageFeature.FEATURE_V_1_3_UNNEST_AND_FLATTEN_ARRAYS,
+                        LanguageFeature.FEATURE_V_1_3_UNPIVOT,
+                        LanguageFeature.FEATURE_V_1_3_WITH_GROUP_ROWS,
+                        LanguageFeature.FEATURE_V_1_3_WITH_RECURSIVE,
+                        LanguageFeature.FEATURE_V_1_4_BARE_ARRAY_ACCESS,
```

Comment:
We should not enable all version 1.4 language features here, only those that BigQuery has. For this PR I'd only add the following, which are the ones I'm absolutely sure BigQuery supports.

This list misses some features which are supported for sure, feel free to add any that you know of. We should closely look at each one and enable them accordingly, but I'd keep that to a separate PR. 

* `FEATURE_V_1_4_ARRAY_AGGREGATION_FUNCTIONS`
* `FEATURE_V_1_4_REMOTE_MODEL`
* `FEATURE_V_1_4_GROUPING_BUILTIN`
* `FEATURE_V_1_4_GROUPING_SETS`
* `FEATURE_V_1_4_GROUP_BY_ALL`